README
NAME
    Bundle::JONASBN - Collection of modules often used by jonasbn

SYNOPSIS
        perl -MCPAN -e 'install Bundle::JONASBN'

CONTENTS
    AppConfig

    CPAN::Mini

    CPAN::Reporter

    Class::Accessor

    Cwd

    DBD::Mock

    DBI

    Data::Dumper

    Data::FormValidator

    Date::Calc

    DateTime

    Devel::Cover

    Error

    ExtUtils::Install, 1.47

    File::Path

    File::Slurp

    Hash::Merge

    JSON

    List::MoreUtils

    Module::Build, 0.35

    Module::Info::File

    Module::Load

    Params::Validate, 0.85

    Perl::Tidy

    Pod::Simple::Wiki

    Spreadsheet::WriteExcel

    Template

    Term::ReadLine::Perl

    Test::Class

    Test::Exception

    Test::Kwalitee

    Test::MockObject::Extends

    Test::More

    Test::Perl::Critic

    Test::Pod, 1.14

    Test::Pod::Coverage, 1.04

    Test::Prereq

    Tie::IxHash

    WWW::Mechanize

    WWW::Mechanize::Cached

    XML::Simple

    YAML

SEE ALSO
    *   Bundle::COG

AUTHOR
    Jonas B. Nielsen (jonasbn), "<jonasbn@cpan.org>"

COPYRIGHT & LICENSE
    Copyright 2009-2014 Jonas B. Nielsen (jonasbn), All Rights Reserved.

    This program is free software; you can redistribute it and/or modify it
    under the same terms as Perl itself.



Changes
Changes file for Bundle::JONASBN

Project site changelog:

http://logicLAB.jira.com/browse/BJONASBN#selectedTab=com.atlassian.jira.plugin.system.project%3Achangelog-panel

0.07 2014-01-20

- Update not required

- Added use of strict and warnings addressing CPANTS/kwalitee rating

- Specified Perl version 5.6.0


0.06 2013-08-02

- Update not required

- Fixed up Changes file as part of my Questhub quest adhering to the standard
  described in: CPAN::Changes::Spec [BJONASBN-51]

  Quest: http://questhub.io/realm/perl/quest/51f0fd8e18ba7dcf4b000040
  Spec: https://metacpan.org/module/CPAN::Changes::Spec
  
  https://logiclab.jira.com/browse/BJONASBN-51
  
- This release will not be merged into trunk since the same work has been
  done for the Task::BeLike::JONASBN, spin off from this distribution
  See series 1.XX
  

0.05 2010-09-04

- Update not required

- This distribution is deprecated and it will be removed from CPAN sometime
  in the future. Please refer to Task::BeLike::JONASBN, which have taken
  this distribution's place


0.04 2010-01-30

- Update required

- Added POD syntax test to test suite

- Addressed failing tests, my perception of a bundle does not seem to be a
real bundle


0.03 2009-10-08

- Update not required

- Added DataFormValidator [BJONASBN-13]

- Added CPAN::Mini [BJONASBN-14] 

- Added CPAN::Reporter [BJONASBN-15] 

- Added Pod::Simple::Wiki [BJONASBN-11] 

- Added DBD:.Mock [BJONASBN-9] 

- Added DateTime [BJONASBN-10] 


0.02 2009-10-05

- Update not required

- Added sorting to Build contents action [BJONASBN-7]

- Added WWW::Mechanize [BJONASBN-2]

- Added WWW::Mechanize::Cached [BJONASBN-3]

- Added Test::MockObject::Extends [BJONASBN-4]

- Added File::Slurp [BJONASBN-5]

- Added Devel::Cover [BJONASBN-6]

- Added Changes file [BJONASBN-8] 


0.01 2009-09-28

- Initial release


Makefile.PL
# Note: this file was auto-generated by Module::Build::Compat version 0.4005
use ExtUtils::MakeMaker;
WriteMakefile
(
  'NAME' => 'Task::BeLike::JONASBN',
  'VERSION_FROM' => 'lib/Task/BeLike/JONASBN.pm',
  'PREREQ_PM' => {
                   'App::Ack' => '0',
                   'App::Prove' => '0',
                   'AppConfig' => '0',
                   'CPAN::Mini' => '0',
                   'CPAN::Reporter' => '0',
                   'Class::Accessor' => '0',
                   'Cwd' => '0',
                   'DBD::Mock' => '0',
                   'DBI' => '0',
                   'Data::Dumper' => '0',
                   'Data::FormValidator' => '0',
                   'Date::Calc' => '0',
                   'DateTime' => '0',
                   'Devel::Cover' => '1.00',
                   'Error' => '0',
                   'ExtUtils::Install' => '1.47',
                   'File::Copy' => '0',
                   'File::Find::Rule' => '0',
                   'File::Path' => '0',
                   'File::Slurp' => '0',
                   'File::Which' => '0',
                   'Hash::Merge' => '0',
                   'JSON' => '0',
                   'List::MoreUtils' => '0',
                   'Module::Build' => '0.35',
                   'Module::Info::File' => '0',
                   'Module::Load' => '0',
                   'Params::Validate' => '0.85',
                   'Perl::Critic' => '0',
                   'Perl::Critic::Bangs' => '0',
                   'Perl::Critic::logicLAB' => '0',
                   'Perl::Tidy' => '0',
                   'Pod::Simple::Wiki' => '0',
                   'Spreadsheet::WriteExcel' => '0',
                   'TAP::Formatter::JUnit' => '0',
                   'Task::Jenkins' => '0',
                   'Template' => '0',
                   'Term::ReadLine::Perl' => '0',
                   'Test::CPAN::Changes' => '0',
                   'Test::Class' => '0',
                   'Test::DatabaseRow' => '0',
                   'Test::Exception' => '0',
                   'Test::Fatal' => '0',
                   'Test::Kwalitee' => '0',
                   'Test::MockObject::Extends' => '0',
                   'Test::More' => '0',
                   'Test::Perl::Critic' => '0',
                   'Test::Pod' => '1.14',
                   'Test::Pod::Coverage' => '1.04',
                   'Test::Prereq' => '0',
                   'Tie::IxHash' => '0',
                   'Try::Tiny' => '0',
                   'WWW::Mechanize' => '0',
                   'WWW::Mechanize::Cached' => '0',
                   'XML::Simple' => '0',
                   'YAML' => '0',
                   'YAML::Shell' => '0',
                   'local::lib' => '0'
                 },
  'INSTALLDIRS' => 'site',
  'EXE_FILES' => [],
  'PL_FILES' => {}
)
;


lib/Bundle/JONASBN.pm
package Bundle::JONASBN;

use strict;
use warnings;

our $VERSION = '0.07';

1;

__END__

=pod

=head1 NAME

Bundle::JONASBN - Collection of modules often used by jonasbn

=head1 SYNOPSIS

    perl -MCPAN -e 'install Bundle::JONASBN'

=head1 CONTENTS

AppConfig

CPAN::Mini

CPAN::Reporter

Class::Accessor

Cwd

DBD::Mock

DBI

Data::Dumper

Data::FormValidator

Date::Calc

DateTime

Devel::Cover

Error

ExtUtils::Install, 1.47

File::Path

File::Slurp

Hash::Merge

JSON

List::MoreUtils

Module::Build, 0.35

Module::Info::File

Module::Load

Params::Validate, 0.85

Perl::Tidy

Pod::Simple::Wiki

Spreadsheet::WriteExcel

Template

Term::ReadLine::Perl

Test::Class

Test::Exception

Test::Kwalitee

Test::MockObject::Extends

Test::More

Test::Perl::Critic

Test::Pod, 1.14

Test::Pod::Coverage, 1.04

Test::Prereq

Tie::IxHash

WWW::Mechanize

WWW::Mechanize::Cached

XML::Simple

YAML

=head1 SEE ALSO

=over

=item * L<Bundle::COG>

=back

=head1 AUTHOR

Jonas B. Nielsen (jonasbn), C<< <jonasbn@cpan.org> >>

=head1 COPYRIGHT & LICENSE

Copyright 2009-2014 Jonas B. Nielsen (jonasbn), All Rights Reserved.

This program is free software; you can redistribute it and/or modify
it under the same terms as Perl itself.

=cut


Build.PL
# Build.PL for Bundle-JONASBN

# $Id$

use strict;
use warnings;

use Module::Build::Bundle;

our $VERSION = '0.01';

my $build = Module::Build::Bundle->new(
    dist_author   => 'Jonas B. Nielsen (jonasbn), <jonasbn@cpan.org>',
    module_name   => 'Bundle::JONASBN',
    license       => 'artistic',
    create_readme => 1,
    requires      => {
        'AppConfig'                 => '0',
        'Class::Accessor'           => '0',
        'CPAN::Mini'                => '0',
        'CPAN::Reporter'            => '0',
        'Cwd'                       => '0',
        'DBD::Mock'                 => '0',
        'DBI'                       => '0',
        'Data::Dumper'              => '0',
        'Data::FormValidator'       => '0',
        'Date::Calc'                => '0',
        'DateTime'                  => '0',
        'Devel::Cover'              => '0',
        'Error'                     => '0',
        'ExtUtils::Install'         => '1.47',
        'File::Path'                => '0',
        'File::Slurp'               => '0',
        'Hash::Merge'               => '0',
        'JSON'                      => '0',
        'List::MoreUtils'           => '0',
        'Module::Build'             => '0.35',
        'Module::Info::File'        => '0',
        'Module::Load'              => '0',
        'Params::Validate'          => '0.85',
        'Perl::Tidy'                => '0',
        'Pod::Simple::Wiki'         => '0',
        'Spreadsheet::WriteExcel'   => '0',
        'Template'                  => '0',
        'Test::Class'               => '0',
        'Test::Exception'           => '0',
        'Test::Kwalitee'            => '0',
        'Test::MockObject::Extends' => '0',
        'Test::More'                => '0',
        'Test::Perl::Critic'        => '0',
        'Test::Pod'                 => '1.14',
        'Test::Pod::Coverage'       => '1.04',
        'Test::Prereq'              => '0',
        'Term::ReadLine::Perl'      => '0',
        'Tie::IxHash'               => '0', #used in dist
        'WWW::Mechanize'            => '0',
        'WWW::Mechanize::Cached'    => '0',
        'XML::Simple'               => '0',
        'YAML'                      => '0',
        'perl'                      => '5.6.0',
    },
    meta_merge => {
        resources => {
            homepage   => 'http://logicLAB.jira.com/browse/BJONASBN',
            repository => 'http://logicLAB.jira.com/svn/BJONASBN',
        }
    },
);

$build->create_build_script();
